angrmanagement/ui/workspace.py

```diff
@@ -108,6 +137,33 @@ def add_view(self, view, caption, category):
         self.dockable_views.append(dock)
         self.view_to_dockable[view] = dock
 
+    def unsplit_view(self):
+        if self.is_split == 1:
+            id = self.split_tab_id
```

Comment:
Nitpick: Do not shadow the builtin function `id`. Use a different variable name.